intakebuilder/getinfo.py

```diff
@@ -84,7 +84,7 @@ def getInfoFromFilename(filename,dictInfo,logger):
 #adding this back to trace back some old errors
 def getInfoFromGFDLFilename(filename,dictInfo,logger):
     # 5 AR: get the following from the netCDF filename e.g. atmos.200501-200912.t_ref.nc
-    if(filename.endswith(".nc")):
+    if(filename.endswith(".nc") and not filename.startswith(".")):
```

Comment:
In the future, let's make a PR for every bug fix or feature. For now, please paste a case where you tested this on where hidden files are skipped.

Reply:
Commented out for now and will hopefully be added later.